# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

navigate_in_rowActionGoal action_goal
navigate_in_rowActionResult action_result
navigate_in_rowActionFeedback action_feedback
